src/components/Infoblock/Infoblock.tsx

```diff
@@ -0,0 +1,183 @@
+import { Box, Button, Container, Stack, Typography } from "@mui/material";
+import { CTA } from "@types";
+import { useEffect, useState } from "react";
+
+export interface InfoblockProps {
+  overline?: string;
+  title: string;
+  content?: string;
+  ctaPrimary?: CTA;
+  ctaSecondary?: CTA;
+  inverse: boolean;
+  image: string;
+  altText?: string;
+  imageShadow: boolean;
+  imageType?: "circle";
+  aspectRatio?: "4/3" | "9/16";
+}
+
+export const Infoblock = ({
+  overline,
+  title,
+  content,
+  ctaPrimary,
+  ctaSecondary,
+  inverse = false,
+  image,
+  altText = "info image",
+  imageShadow,
+  imageType,
+  aspectRatio = "4/3",
+}: InfoblockProps) => {
+  const [imageTypePattern, setImageTypePattern] = useState("");
+
+  const left = aspectRatio === "4/3" ? "11%" : "17%";
```

Comment:
Try to avoid magic numbers. Use spacing values from theme, instead. If obligated, be consistent (use same spacing values for the various margins)